Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -58,8 +58,10 @@ spec:
         image: registry.access.redhat.com/rhel7/sssd:latest
 
         sssdConfigFile:
-          configMap:
-            name: my-nfs-sssd-config
+          volumeSource:
+            configMap:
+              name: my-nfs-sssd-config
+              defaultMode: 0600 # mode must be 0600
```

Comment:
maybe a comment  why mode has to be 0600?

Reply:
It's not uncommon for config files to have to have strict permissions, so I'm not sure how helpful the reason is.